spec/requests/notifications_controller_spec.rb

```diff
@@ -172,24 +172,6 @@ def delete_notification(resp_code, matcher)
             )
           end
 
-          it "gets notifications list with unread high priority notifications at the top when navigation menu is legacy" do
```

Comment:
This test relates to the navigation_menu, not the notifications menu. So I don't think we should remove it as part of this PR?